Add GPU trigger project

dcraik opened the pull request.

dcraik committed:
updated presentations for dcraik

dcraik committed:
Merge branch 'master' of https://github.com/iris-hep/iris-hep.github.io-source

dcraik committed:
Updated talks

dcraik committed:
merge

dcraik committed:
Add GPU trigger project

davidlange6 commented:
hi @dcraik - please add some pointers to the main Allen project pages and collaboration. Eg, we should not appear as being the sole contributors

dcraik committed:
Update Allen

dcraik commented:
> hi @dcraik - please add some pointers to the main Allen project pages and collaboration. Eg, we should not appear as being the sole contributors

Hi @davidlange6 - as far as I'm aware, there isn't yet a project page for Allen but I've changed "developing" to "participating in the development of" and added links to the git repository and paper, which both make clear the wider collaboration.

dcraik committed:
Add Allen publication

henryiii commented on pages/projects/allen.md:
```diff
@@ -0,0 +1,18 @@
+title: GPU Trigger Project
+shortname: allen
+pagetype: project
+image: logos/allen.png
```
```suggestion
image: logos/allen.png
logowidth: 50%
```

This will show the logo on the top of the project page (adjust width if needed).

henryiii commented on pages/projects/allen.md:
```diff
@@ -0,0 +1,18 @@
+---
+
+We are participating in the development of a GPU-based trigger, which would replace the first-level trigger in the LHCb upgrade. 
+A paper describing the project can be found [here](https://arxiv.org/abs/1912.09161) and the code is available in [the gitlab repository](https://gitlab.cern.ch/lhcb/Allen).
```
Can't you just add this as a paper in `/_data/publications` and mark it with `project: allen`? Then it will show up on this page automatically.

henryiii replied:
Edit: Looks like you added the publication so the part about the paper here probably should be removed.
```suggestion
The code is available in [the gitlab repository](https://gitlab.cern.ch/lhcb/Allen).
```

henryiii committed:
Apply suggestions from code review
